packages/datahub-data/migrations/meta/_journal.json

```diff
@@ -0,0 +1,20 @@
+{
+  "version": "5",
+  "dialect": "mysql",
+  "entries": [
+    {
+      "idx": 0,
+      "version": "5",
+      "when": 1694789225065,
+      "tag": "0000_soft_vengeance",
```

Comment:
Do the `tag` names have meaning?

Reply:
This is the generated name of the migration. I renamed it by renaming the file, but I also need to rename it here. I cannot find a way to generate a migration with a meaningful name. So, if we want to have a meaningful name, we need to rename it by hand and remember to rename it in this file.

Reply:
The rest of the code also calls it an ID. For example, in the function name and response of `heritageObjects.getById`.

Reply:
Decided in meeting: We will not use the generated names. For now, we will change the names by hand, but it would be nice to have a script for that.